lib/oneview-sdk/resource/api200/uplink_set.rb

```diff
@@ -78,6 +85,13 @@ def add_fcoenetwork(network)
         raise IncompleteResource, 'Must set network uri attribute' unless uri
         @data['fcoeNetworkUris'].push(uri)
       end
+
+      # Gets the unassigned uplink ports
+      def get_unassigned_ports
```

Comment:
In the Oneview UI, when I will create a new uplink to a logical interconnect, a request is did using that URI to get the unassigned ports.